smart-contract/README.md

```diff
@@ -308,7 +308,7 @@ Grant for bank 1
 provenanced tx marker grant-authz \
     tp14hj2tavq8fpesdwxxcu44rty3hh90vhuz3ljwv \
     "transfer" \
-    --transfer-limit 50000000usdf.local,50000000bank1.coin,50000000bank2.coin \
```

Comment:
member bank can no longer redeem for other bank coins so authz only needs to include usdf and bank member reserve token